workshops/test/test_util.py

```diff
@@ -70,6 +70,47 @@ def test_serializability_of_parsed(self):
             self.fail('Dumping person_tasks to JSON unexpectedly failed!')
 
 
+    def test_no_new_column_added(self):
+        ''' Ensure more number of fields are not added by increasing the number of columns'''
+        csv = """personal,middle,family,email,event,role,phone_no
+john,a,doe,johndoe@email.com,2015-09-01-ttt-online,learner,9991239011
+jane,a,doe,janedoe@email.com,2015-09-01-ttt-online,learner,9991239021"""
+        person_tasks, _ = self.compute_from_string(csv)
+        person = person_tasks[0]
+        self.assertTrue('phone_no' not in person)
+
+    def test_number_of_columns(self):
+        ''' Ensure number of columns remains same in case of empty fields or extra fields '''
+
+        csv = """personal,middle,family,email,event,role,phone_no
+john,a,doe,johndoe@email.com,2015-09-01-ttt-online,learner,9991239011"""
+        person_tasks, _ = self.compute_from_string(csv)
+        person = person_tasks[0]
+        self.assertEqual(len(person),7)
```

Comment:
Instead of checking for number of items, I'd rather check if the `person.keys()` corresponds to `Person.PERSON_TASK_UPLOAD_FIELDS` - either by hardcoding field names, or by comparing directly with `Person.PERSON_TASK_UPLOAD_FIELDS`.

The same applies to other `self.assertEqual(len(person),7)` in this test.